multiapps-controller-core/src/test/resources/mta/devxwebide/xs2-apps.json

```diff
@@ -20,8 +20,12 @@
     "staging": {
       "buildpacks": []
     },
-    "uris": [
-      "webide.sofd60245639a"
+    "routes": [
+      {
+        "domain": "sofd60245639a",
```

Comment:
If it is not too mu effort remove theses internal entities like sofd60245639a and i users

Reply:
this should probably be a separate task, as we have a lot of json test files with these, I think

Reply:
Good